source/outlines/2016-03-10.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: 20160310
+layout: outline
+---
+
+## All
+
+### Word of the Day
+
+* bowdlerize - transitive verb - to remove or modify the parts (of a book,
+for example) considered offensive.
+
+
+## 1508
+
+### Portfolio Reviews (9:00 - WE FINISH)
+
+In this order:
```

Comment:
How do you feel about randomizing this?

```
 * Matt Rooney
 * Torie Warren
 * Emily Dowdle
 * Ryan Johnson
 * Adam Jensen
 * Marlo Major
 * Jhun de Andres
 * Ross Edfort
 * Amber Crawford
 * Jessica Goulding
 * Jill Donohue
 * Tyler Komoroske
 * Pat Wey
 * Matt Stjernholm
 * Aaron Careaga
 * Sekhar Paladugu
 * Jason Pilz
 * Robbie Lane
```

That was the random list my randomizer made...